### PR TITLE
Feature/fpay 1394

### PR DESCRIPTION
**Pull request by jaroslawgabara**

### CODE REVIEW

File: dev-aws/kafka-shared-msk/payment-platform/payment.tf [hunk 1]
```diff
@@ -76,10 +76,30 @@ resource "kafka_topic" "payment_v1_public_events_cbc_topup_v3" {
   }
 }
 
+//All public events will go here. It's meant to be used by ops downstream.
```
Comment (sbuliarca) [inline dev-aws/kafka-shared-msk/payment-platform/payment.tf:79]: Please fix the linting issues

File: dev-aws/kafka-shared-msk/payment-platform/payment.tf [hunk 2]
```diff
@@ -76,10 +76,30 @@ resource "kafka_topic" "payment_v1_public_events_cbc_topup_v3" {
...
+//It should not be consume by a particular domain team.
+resource "kafka_topic" "payment_v1_public_events" {
+  name               = "payment-platform.payment.v1.public.events"
+  replication_factor = 3
+  partitions         = 5
```
Comment (sbuliarca) [inline dev-aws/kafka-shared-msk/payment-platform/payment.tf:84]: Just checking: is this topic migrated from an existing cluster? If true, it should have exactly the same number of partitions.

Comment (jaroslawgabara) [reply dev-aws/kafka-shared-msk/payment-platform/payment.tf:84]: nope. it's a new topic

File: dev-aws/kafka-shared-msk/payment-platform/payment.tf (hunk 1 above)
Comment (jaroslawgabara) [reply dev-aws/kafka-shared-msk/payment-platform/payment.tf]: done

**Review by sbuliarca — APPROVED** [review]

**Pull request merged**